distrobox-init

```diff
@@ -275,6 +275,21 @@ if ! command -v mount || ! command -v mount || ! command -v passwd ||
 			shadow \
 			sudo \
 			util-linux
+	elif command -v emerge; then
+		# set the password requirements to the minimum. gentoos passwd policy is stricter than other distros
+		# otherwise setting up the container can fail.
+		# this is needed on ubuntu 21.10
+		# works by setting /etc/security/passwdqc.conf to min=1,1,1,1,1 + everything except the first line from
```

Comment:
is this mandatory on gentoo?
A new approach now is to initialize user's password to a value and then delete the password, gentoo should allow that without problems right?

From your comments seems like it's needed also on Ubuntu 21.10+ but I've been using them daily no problems

Reply:
The new approach works for me. I misunderstand the code and thought distrobox uses the host password, please ignore the comment. It was needed when initializing the Gentoo container, not when using Ubuntu 21.10.